packages/manager/modules/iam/src/iam.constants.js

```diff
@@ -31,49 +31,111 @@ const ENTITY = {
 
 const ENTITY_NAME_PATTERN = /^[a-zA-Z0-9-/_+]*$/;
 
-// TODO wait for other guide specifications
 const GUIDE = {
   IAM: {
-    FR: 'https://docs.ovh.com/fr/customer/IAM/',
+    DEFAULT:
+      'https://help.ovhcloud.com/csm/en-ie-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056806',
+    ASIA:
+      'https://help.ovhcloud.com/csm/asia-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056797',
```

Comment:
Add India specific link(same as ASIA or IN specific)
```suggestion
    ASIA:
      'https://help.ovhcloud.com/csm/asia-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056797',
      IN:
      'https://help.ovhcloud.com/csm/asia-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056797',
```